docs/tutorials/update-task-description.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+---
+
+# Update a task description
+
+In this tutorial, you will learn how to use Postman to update a task description. This use case is common, since your users will need to to add task details as their deadlines approach. 
+
+This tutorial takes about 15 minutes to complete. 
+
+## Before you begin
+
+Before starting this tutorial, install Postman and json-server. Then, sync the To-Do service to your  GitHub repository so that you can clone it to your desktop. For a complete set of steps, see Before you start a tutorial. 
+
+> ⚠️ The steps to run json-server are designed for Mac users, but you can easily adapt them to Windows or Linux. For example, on Windows, replace `json-server.sh` with `json-server.bat` from the source repository.
+
+## Get a list of tasks
+
+Unless you know the `id` of the task you want to update, start with listing all tasks in the To-Do service's database. 
+
+##### To get a list of tasks:
+
+1. In Terminal, start json-service. 
+
+    ```shell
+    cd path/to/<your-github-workspace>/to-do-service/api
+    # Run the service against the database file
+    ./json-server.sh -w to-do-db-source.json
+    ```
+
+2. In Postman, add a new request with these values:
+   
+    * **METHOD** — `GET`
+    * **URL** — `{localhost}:{3000}/tasks` 
+    * **Headers** — `Content-Type: application/json`
+    * **Request body** — None
+
+3. In Postman, click **Send**.
+   The To-Do service returns a JSON object that contains all tasks. Its response looks like the following example. Note the `id` of the task to update. 
+
+    ```json
+    [
+        {
+            "task_title": "My new task",
+            "task_description": "My example description",
+            "id": "1"
+        },
+        {
+            "task_title": "My new task",
+            "task_description": "My example description",
+            "id": "2"
+        },
+    ]
+    ```
```

Comment:
These examples don't match the documentation of the task resource: [task](https://uwc2-apidoc.github.io/to-do-service-public/api/task). Can you update them to match, please?